### PR TITLE
[frontend] Fix filtering for toolbar

### LINKED ISSUE #6411: "Select all" selects the whole platform, not just what's on the screen
## Description

When I do a "select all" to delete relations, when I look at my background task, I realize that it contains 160K elements. That's not what I selected at the beginning.

## Environment

OCTI 6.0.7

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Got to a threat actor group
2. Go to knowledge > incidents (if you don't have incident here, add one)
3. Go to the "relation view" 
4. Select all and click on the bin in the bottom right corner to remove the relation.
5. Launch the background task and check the amount of entity that is processed.

### PR DESCRIPTION
**Pull request by Kedae**
Closes #6411

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/views/EntityStixCoreRelationshipsRelationshipsView.tsx [hunk 1]
```diff
@@ -143,7 +143,18 @@ const EntityStixCoreRelationshipsRelationshipsView: FunctionComponent<EntityStix
...
+    filters: [
+      ...predefinedFilters,
+      {
+        key: 'regardingOf',
+        operator: 'eq',
+        mode: 'and',
+        values: [
+          { key: 'id', values: [entityId], operator: 'eq', mode: 'or' },
+          { key: 'relationship_type', values: relationshipTypes, operator: 'eq', mode: 'or' },
+        ] as unknown as string[], // Workaround for typescript waiting for better solution
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/views/EntityStixCoreRelationshipsRelationshipsView.tsx]: This means we have to define different cases for values depending on the key.
I can have a look if you need help.

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/views/EntityStixCoreRelationshipsRelationshipsView.tsx]: 'in regards of' is a filter for entities, not relationships

**Review by Archidoit — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/data/ToolBar.jsx [hunk 2]
```diff
@@ -1645,9 +1646,7 @@ class ToolBar extends Component {
                                     )}
                                   </span>
                                 )}
-                                <TasksFilterValueContainer
-                                  filters={filters}
-                                ></TasksFilterValueContainer>
+                                <TasksFilterValueContainer filters={filters} entityTypes={entityTypes}></TasksFilterValueContainer>
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/data/ToolBar.jsx]:
```suggestion
                                <TasksFilterValueContainer filters={filters} entityTypes={entityTypes} />
```

File: opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx [hunk 3]
```diff
@@ -258,6 +258,7 @@ const TasksList = ({ data }) => {
                         && (isFilterGroupNotEmpty(filters)
                           ? <TasksFilterValueContainer
                               filters={filters}
+                              entityTypes={['Stix-Core-Object', 'stix-core-relationship', 'Notification']}
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx:261]: I see heterogeneous casing. We don' need capital letters ?

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx:261]: stix-core-relationship is not in capital in this case... yes that's heterogeneous :(

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx:261]: we need to refacto all the occurences if we want to homogenize...

**Review by labo-flg — APPROVED** [review]
Tested OK

**Pull request merged**